src/test/java/cloud/fogbow/ras/core/plugins/interoperability/cloudstack/volume/v4_9/CloudStackVolumePluginTest.java

```diff
@@ -94,6 +101,474 @@ public void setUp() throws UnexpectedException, InvalidParameterException {
+    // It returns the same name of the parameter.
+    @Test
+    public void testNormalizeInstanceNameWhenParameterIsNotNull() {
+        // set up
+        String nameExpected = "nameExpected";
+
+        // exercise
+        String name = this.plugin.normalizeInstanceName(nameExpected);
+
+        // verify
+        Assert.assertEquals(nameExpected, name);
+    }
+
+    // test case: When calling the buildVolumeCustomized method, it must verify if
+    // It returns a right CreateVolumeRequest.
+    @Test
+    public void testBuildVolumeCustomizedSuccessfully() throws InvalidParameterException {
+        // set up
+    String diskOfferingId = "diskOfferingId";
+        String nameExpected = "nameExpected";
+        int volumeSizeExpected = 1;
+        VolumeOrder volumeOrder = Mockito.mock(VolumeOrder.class);
+        Mockito.when(volumeOrder.getName()).thenReturn(nameExpected);
+        Mockito.when(volumeOrder.getVolumeSize()).thenReturn(volumeSizeExpected);
+
+        CreateVolumeRequest requestExpected = new CreateVolumeRequest.Builder()
+                .zoneId(this.zoneId)
+                .name(nameExpected)
+                .diskOfferingId(diskOfferingId)
+                .size(String.valueOf(volumeSizeExpected))
+                .build(this.cloudStackUrl);
+
+        // exercise
+        CreateVolumeRequest request = this.plugin.buildVolumeCustomized(volumeOrder, diskOfferingId);
+
+        // verify
+        Assert.assertEquals(requestExpected.getUriBuilder().toString(), request.getUriBuilder().toString());
+    }
+
+    // test case: When calling the getDiskOfferingIdCustomized method and the is no disk offering
+    // customized with the size required, it must verify if It returns a null.
+    @Test
+    public void testGetDiskOfferingIdCustomizedWhenNotFoundDiskOffering() {
+        // set up
+        List<GetAllDiskOfferingsResponse.DiskOffering> disksOffering = new ArrayList<>();
+        int diskSizeOne = CloudStackVolumePlugin.CUSTOMIZED_DISK_SIZE_EXPECTED;
+        String diskOfferingIdOne = "idOne";
+        boolean isNotCustomized = false;
+        GetAllDiskOfferingsResponse.DiskOffering diskOfferingOne =
+                Mockito.mock(GetAllDiskOfferingsResponse.DiskOffering.class);
+        Mockito.when(diskOfferingOne.getId()).thenReturn(diskOfferingIdOne);
+        Mockito.when(diskOfferingOne.getDiskSize()).thenReturn(diskSizeOne);
+        Mockito.when(diskOfferingOne.isCustomized()).thenReturn(isNotCustomized);
+        int diskSizeTwo = CloudStackVolumePlugin.CUSTOMIZED_DISK_SIZE_EXPECTED;
+        String diskOfferingIdTwo = "idTwo";
+        GetAllDiskOfferingsResponse.DiskOffering diskOfferingTwo =
+                Mockito.mock(GetAllDiskOfferingsResponse.DiskOffering.class);
+        Mockito.when(diskOfferingTwo.getId()).thenReturn(diskOfferingIdTwo);
+        Mockito.when(diskOfferingTwo.getDiskSize()).thenReturn(diskSizeTwo);
+        Mockito.when(diskOfferingTwo.isCustomized()).thenReturn(isNotCustomized);
```

Comment:
This code snippet seems to be repeated in more than one test, perhaps it can be moved to a method or procedure in some of the test helper classes to minimize the complexity of these tests.

Reply:
Done.
Note: I've done the changes in the fourth PR in order to avoid conflicts.